ecl/ecl-package/CMakeLists.txt

```diff
@@ -56,7 +56,12 @@ include_directories (
 ADD_DEFINITIONS( -D_CONSOLE )
 
 HPCC_ADD_EXECUTABLE ( ecl-packagemap ${SRCS} )
-add_dependencies ( ecl-packagemap espscm ws_packageprocess )
+
+set( WS_PACKAGEPROCESS_DEP )
+if( NOT CLIENTTOOLS_ONLY )
```

Comment:
I think the dependency here is just wrong.

@afishbeck Do you know anything about this?

I not also that there are similar issues in 

./ecl/eclcmd/queries/CMakeLists.txt:add_dependencies ( ecl-queries espscm ws_workunits )
./ecl/eclcmd/roxie/CMakeLists.txt:add_dependencies ( ecl-roxie espscm ws_workunits )